crates/holochain/src/conductor/state.rs

```diff
@@ -200,6 +201,18 @@ impl ConductorState {
             .values()
             .find(|app| app.all_cells().any(|id| id == cell_id))
     }
+
+    /// Get network compability params
+    /// (but this can't actually be on the Conductor since it must be retrieved before
+    /// conductor initialization)
+    pub fn get_network_compat(&self) -> NetworkCompatParams {
+        NetworkCompatParams {
+            dpki_hash: {
+                tracing::warn!("Using default NetworkCompatParams");
```

Comment:
does that actually work having a tracing line amid a struct literal? my brain hurts

Reply:
Haha, it's just a block, and the value of the block is always the last line. So yes. Wanted the warning as close as possible to what it's about.

Reply:
:upside_down_face: